ding2.install

```diff
@@ -327,3 +306,10 @@ function ding2_update_7014() {
 function ding2_update_7015() {
   module_enable(array('search_api', 'search_api_multi', 'search_api_db', 'search_api_views'));
 }
+
+/**
+ * Enable search_api modules.
+ */
+function ding2_update_7016() {
+  module_enable(array('ddb_cp', 'environment_indicator'));
```

Comment:
According to https://github.com/ding2/ding2/blob/master/modules/ddb_cp/README.md ddb_cp is only relevant for DBC hosted sites. Many sites are not.